### ISSUE TITLE
Is "toFixed(5)" conversion need?

### ISSUE THREAD

**Issue by jongmoon** [naver/egjs-axes#116]
## Description

There are `toFixed(5)` conversion on `Coordinates.ts`. 

https://github.com/naver/egjs-axes/blob/69c8c9d67ea9f0053259d75840edf67542c68439/src/Coordinate.ts#L18

https://github.com/naver/egjs-axes/blob/69c8c9d67ea9f0053259d75840edf67542c68439/src/Coordinate.ts#L48

because of this reason, code that depends on axes must convert float as `toFixed(5) format`. It is annoying job. I don't know why this conversion is used.

**Comment by jongmoon** [comment]
@younkue thinks it is because of `0.1 + 0.2` is not `0.3`

![image](https://user-images.githubusercontent.com/3255471/47403322-1737fc00-d784-11e8-8464-32df74ebb50c.png)

**Comment by jongmoon** [comment]
**I found the case `toFixed(5)` makes bug.**

For example, If I set value `58.52194572964862` which has very many floating point.

But when it is animated, it's destPos is changed by `toFixed(5)`.
so its new destPos is changed to `58.521946`

*toFixed(5) is occurs in getCirculatedPos in following code.*
https://github.com/naver/egjs-axes/blob/52e825411d59ab58fa17899d8bd49b75fac65118/src/AnimationManager.ts#L198

So although it's not changed, I assume it's changed somewhat(0.000004999999994481641)

And it trigger `change` event! it is far from our intend.
